test/util/utils.go

```diff
@@ -90,6 +90,19 @@ func MakePG(name, namespace string, min int32, creationTime *time.Time, minResou
 	return pg
 }
 
+func UpdatePGStatus(pg *v1alpha1.PodGroup, phase v1alpha1.PodGroupPhase, occupiedBy string, scheduled int32, running int32, succeeded int32,
```

Comment:
this formatting is bugging me you should either keep all params in one line or split them and keep only one per line. You can also think about just passing status to this function instead of all possible params, like this:
```go
func UpdatePGStatus(pg *v1alpha1.PodGroup, status v1alpha1.PodGroupStatus) *v1alpha1.PodGroup{
    pg.Status = status
    return pg
}
```